std/process.d

```diff
@@ -1280,7 +1291,12 @@ private Pid spawnProcessImpl(scope const(char)[] commandLine,
     if (!CreateProcessW(null, commandLine.tempCStringW().buffPtr,
                         null, null, true, dwCreationFlags,
                         envz, workDir.length ? pworkDir : null, &startinfo, &pi))
-        throw ProcessException.newFromLastError("Failed to spawn new process");
+    {
+        if (GetLastError() == ERROR_FILE_NOT_FOUND)
+            throw new ProcessException(text("Executable file not found: ", program));
```

Comment:
Just noticed that this error message has the exact same wording as for when `searchPathFor` failed. We should probably avoid having errors that share the message string but indicate different error conditions.

Reply:
Yeah, that's what I meant by 'consistent with the Posix msg' (`searchPathFor` is Posix only).

Reply:
OK. Sorry, at the time I did not know what exactly you meant by that. I understand that the two can refer to different error conditions (a file name was not found in `$PATH` vs. an absolute file name does not exist on disk).

Reply:
Can be relative too. FWIW, I think Posix resulting in 2 different msges for the same error (program not found) isn't ideal; to me it doesn't matter whether the program contains a slash or not.

Reply:
> to me it doesn't matter whether the program contains a slash or not.

What do you mean? Whether the path contains a path separator or not is what decides if the `PATH` variable is consulted. The `Posix` logic mirrors the Windows API logic (quote "If the file name does not contain a directory path"), to make the Phobos API consistent across both platforms.